### PR TITLE
Set deployment target to iOS 9

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues: 

* [FH-3927](https://issues.jboss.org/browse/FH-3927) - Set deployment target to iOS 9 on blank-ios-app
* [FH-3950](https://issues.jboss.org/browse/FH-3950) - Move blank-ios-app from Travis CI to CircleCI

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
 project 'blank-ios-app.xcodeproj'
-platform :ios, '8.0'
+platform :ios, '9.0'
 
 target 'blank-ios-app' do
 	pod 'FH', '~> 3.1.1'
```
Comment (jcesarmobile) [inline Podfile]: This should be updated to 4.0.0

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Deployment target is set to iOS 9 for the project, but not for the target
Also we should use fh-ios-sdk 4.0.0

File: Podfile (hunk 1 above)
Comment (danielpassos) [reply Podfile]: Done!

**Comment by jcesarmobile** [comment]
@danielpassos you missed the "Deployment target is set to iOS 9 for the project, but not for the target" part

**Comment by danielpassos** [comment]
@jcesarmobile I was testing if you are really reviewing it :joy:

**Review by jcesarmobile — APPROVED** [review]

**Pull request merged**